aws-ssm-patchbaseline/template.yml

```diff
@@ -19,4 +19,4 @@ Resources:
     Properties:
       Handler: software.amazon.ssm.patchbaseline.HandlerWrapper::testEntrypoint
       Runtime: java8
-      CodeUri: ./target/aws-ssm-patchbaseline-handler-1.0-SNAPSHOT.jar
+      CodeUri: ./target/aws-ssm-patchbaseline-handler-1.0-SNAPSHOT.jar
```

Comment:
nothing new changed to this file as well, please undo the change. so it won't show up as new file

Reply:
readHandler would definitely fail if you stipe down the return baseline Id

Reply:
this files shouldn't be in Files Changed, as nothing changed

Reply:
yes we dont have any changes and we have reverted to previous version

Reply:
The file has reverted to previous stable changes.